src/v2/Components/LoginSignUpBanner/Banner.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import { Flex, color } from "@artsy/palette"
+import { BannerButton } from "./Button"
+import styled from "styled-components"
+
+interface Props {}
+
+const Container = styled(Flex)`
+  width: 100%;
+  padding: 12px;
+  background-color: ${color("black100")};
+`
+
+export class Banner extends React.Component<Props> {
```

Comment:
Can we use React funcction components instead of classes? (All class use should be considered depreciated, since we can't combine classes and react hooks)